pylti/tests/test_common.py

```diff
@@ -76,7 +78,8 @@ def test_ltioauthdatastore(self):
         self.assertIsNone(store.lookup_consumer("keyNS"))
         self.assertIsNone(store.lookup_cert("keyNS"))
 
-    def test_ltioauthdatastore_no_consumers(self):
+    def test_lti_oauth_data_store_no_consumers(self):
+        # pylint: disable=invalid-name
```

Comment:
@carsongee done with it. but pylint is still giving me invalid-name error, so i disable this messages for these specific functions